### PR TITLE
chore: improve readability of logs

### PR DESCRIPTION
**Pull request by tobymao**

### CODE REVIEW

File: sqlmesh/core/model/definition.py [hunk 1]
```diff
@@ -1158,9 +1157,6 @@ def _data_hash_values(self) -> t.List[str]:
 
         return data
 
-    def __repr__(self) -> str:
```
Comment (izeigerman) [inline sqlmesh/core/model/definition.py:1161]: Why all model-related reprs have been removed?

File: sqlmesh/core/model/definition.py [hunk 2]
```diff
@@ -1330,9 +1326,6 @@ def _data_hash_values(self) -> t.List[str]:
             data.append(column_hash)
         return data
 
-    def __repr__(self) -> str:
-        return f"Model<name: {self.name}, seed: {self.kind.path}>"
```
Comment (izeigerman) [inline sqlmesh/core/model/definition.py:1334]: I think this is still useful

File: sqlmesh/core/node.py [hunk 3]
```diff
@@ -181,10 +182,15 @@ class _Node(PydanticModel):
...
     tags: t.List[str] = []
     stamp: t.Optional[str] = None
+    _path: Path = Path()
 
     _croniter: t.Optional[CroniterCache] = None
     __inferred_interval_unit: t.Optional[IntervalUnit] = None
 
+    def __str__(self) -> str:
+        path = f": {self._path.name}" if self._path else ""
+        return f"{self.__class__.__name__}<{self.name}{path}>"
```
[the remark this answers is not in the document]
Comment (izeigerman) [reply sqlmesh/core/node.py:192]: Please note that `path` is not available after reading from state

File: sqlmesh/core/model/definition.py (hunk 1 above)
Comment (tobymao) [reply sqlmesh/core/model/definition.py:1161]: pydantic

File: sqlmesh/core/snapshot/definition.py [hunk 4]
```diff
@@ -299,6 +311,20 @@ class SnapshotTableInfo(PydanticModel, SnapshotInfoMixin, frozen=True):
...
+        "version",
+        "physical_schema",
+        "change_category",
+        "kind_name",
+    )
+
+    def __str__(self) -> str:
+        args = ", ".join(f"{k}: {getattr(self, k)}" for k in self.REPR_FIELDS)
+        temp_version = f", {self.temp_version}" if self.temp_version else ""
+        return f"SnapshotInfo<{args}{temp_version}>"
```
Comment (izeigerman) [inline sqlmesh/core/snapshot/definition.py:326]: We should use the original class name. `SnapshotInfo` refers to an entity which doesn't exist

File: sqlmesh/core/model/definition.py (hunk 2 above)
Comment (tobymao) [reply sqlmesh/core/model/definition.py:1334]: still have it

File: sqlmesh/core/snapshot/definition.py [hunk 5]
```diff
@@ -119,11 +125,17 @@ def snapshot_id(self) -> SnapshotId:
     def __lt__(self, other: SnapshotId) -> bool:
         return self.name < other.name
 
+    def __str__(self) -> str:
+        return f"{self.name}: {self.identifier}"
```
[the remark this answers is not in the document]
Comment (izeigerman) [reply sqlmesh/core/snapshot/definition.py:129]: IMHO I'd rather not invent new formats and just rely on pydantic.

File: sqlmesh/core/model/definition.py (hunk 2 above)
Comment (izeigerman) [reply sqlmesh/core/model/definition.py:1334]: Can you perhaps show an example of how this looks like for seed models now?

File: sqlmesh/core/node.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (tobymao) [reply sqlmesh/core/node.py:192]: how is it not using path?

[the remark this answers is not in the document]
Comment (izeigerman) [reply sqlmesh/core/node.py:192]:
```
f"{self.__class__.__name__}<{self.name}{path}>"
```
Path is missing curly braces and hence is not interpolated, right?

**Review by izeigerman — APPROVED** [review]

**Pull request merged**